project-base/app/tests/FrontendApiBundle/Functional/Product/Flag/FlagTest.php

```diff
@@ -118,9 +118,6 @@ public function testFlagByUuidFilteredByAnotherFlag(): void
             [
                 'name' => t('32" Philips 32PFL4308', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $this->getFirstDomainLocale()),
             ],
-            [
-                'name' => t('32" Hyundai 32PFL4400', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $this->getFirstDomainLocale()),
```

Comment:
Ah, I see, that was tricky :sweat_smile: So the product was renamed to `Television Hyundai [M]` so it is now alphabetically sorted "later" in the array but the query has a limit for the first 5 products so it "disappeared" from the array. 

Could you please tweak the test itself in a separate commit?
- I would modify the `$products` array so it is of a size of the given limit (5 items) - the rest is just confusing
- also, instead of using hardcoded flag IDs, I would use the persistent references (in the whole `FlagTest`, please)
![image](https://github.com/user-attachments/assets/364f565b-19f9-4bc7-bf50-ee7b318efddf)
- if we want to have it super tuned, we could also move the queries from strings into separate graphql files :smile: 

If you do not want to postpone this task with the refactorings, I am ok with that, just let me know, I will create a separate task for that :wink: